test/test.hpp

```diff
@@ -278,7 +281,8 @@ namespace tts
 
     // Add garbage at the end of sub-native registers
     // For emulated type, there is no such gap so we don't do anything
-    if constexpr( (W::size() < eve::fundamental_cardinal_v<v_t>) && !eve::has_emulated_abi_v<W> )
+    if constexpr( (W::size() < eve::fundamental_cardinal_v<v_t>)&&!eve::has_emulated_abi_v<W>
+                  && eve::current_api != eve::rvv )
```

Comment:
I'm not sure what's happening here and it doesn't strike me as correct.

Reply:
Yep, you are right - after change of `is_full` it RISC-V tags this change is not needed, so removing this.

Earlier I supposed that `is_full` should refer to filling max available physical registers. Because of this, types corresponding for `W data` and ` ftype these` will not be the same, so we can not construct `ftype` from `storage` of `data`.

Now, as `is_full` implemented in terms of intrinsics types,  we are able to construct `ftype` with `storage`.